src/main/java/pl/pw/mini/minispace/dtos/friend/EventInvitationDto.java

```diff
@@ -9,6 +9,6 @@
 public class EventInvitationDto {
     private Long Id;
     private Long eventId;
-    private UserDto fromUserId;
```

Comment:
nie wiem skąd to się wzięło, czy to przez nieaktualne repo? Sam tego nie zmieniałem

Reply:
jakiś przypadek, w developie jest już poprawione na longi